src/query.rs

```diff
@@ -468,4 +522,30 @@ mod tests {
 
         assert!(res.is_err());
     }
+
+    #[test]
+    fn it_can_query_associators() {
+        let wmi_con = wmi_con();
+
+        #[derive(Deserialize, Debug)]
+        struct Win32_DiskDrive {
+            __Path: String,
+            Caption: String,
+        }
+
+        #[derive(Deserialize, Debug)]
+        struct Win32_DiskPartition {
+            Caption: String,
+        }
+
+        let disk = wmi_con.get::<Win32_DiskDrive>().unwrap();
+
+        let results = wmi_con.associators::<Win32_DiskPartition>(&disk.__Path).unwrap();
```

Comment:
I think __Path should be public, otherwise this probably wont work if struct is defined in another module?

Reply:
I'm going to leave it like this, because it simplify the examples and any rust user will not have a problem fixing it if they need to